File: frontend/lib/models/prescription/upload_prescription_response.dart
class UploadPrescriptionResponse {
  final int? prescriptionId;
  final String? filename;
  final String status;
  final String message;

  UploadPrescriptionResponse({
    required this.prescriptionId,
    required this.filename,
    required this.status,
    required this.message,
  });

  factory UploadPrescriptionResponse.fromJson(
      Map<String, dynamic> json) {
    return UploadPrescriptionResponse(
      prescriptionId: json["prescriptionId"],
      filename: json["filename"],
      status: json["status"] ?? "",
      message: json["message"] ?? "",
    );
  }
}

File: backend/src/main/java/com/smarthealthcare/backend/prescription/dto/UploadPrescriptionResponse.java
package com.smarthealthcare.backend.prescription.dto;

public class UploadPrescriptionResponse {

    private Long prescriptionId;
    private String imageUrl;
    private String status;
    private String message;

    public UploadPrescriptionResponse() {
    }

    public UploadPrescriptionResponse(
            Long prescriptionId,
            String imageUrl,
            String status,
            String message) {

        this.prescriptionId = prescriptionId;
        this.imageUrl = imageUrl;
        this.status = status;
        this.message = message;
    }

    public Long getPrescriptionId() {
        return prescriptionId;
    }

    public void setPrescriptionId(Long prescriptionId) {
        this.prescriptionId = prescriptionId;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}